application/views/themeOptions/availablethemelist.php

```diff
@@ -12,29 +12,31 @@
             <div id="templates_no_db" >
                 <table class="items table table-hover">
                     <thead>
-                    <tr>
-                        <th><?php eT('Preview'); ?></th>
-                        <th><?php eT('Folder'); ?></th>
-                        <th><?php eT('Description'); ?></th>
-                        <th><?php eT('Type'); ?></th>
-                        <th><?php eT('Extends'); ?></th>
-                        <th></th>
-                    </tr>
+                        <tr>
+                            <th><?php eT('Preview'); ?></th>
+                            <th><?php eT('Folder'); ?></th>
+                            <th><?php eT('Description'); ?></th>
+                            <th><?php eT('Type'); ?></th>
+                            <th><?php eT('Extends'); ?></th>
+                            <th></th>
+                        </tr>
                     </thead>
 
-            <tbody>
-            <?php foreach ($aQuestionThemes['available_themes'] as $oTheme): ?>
-                <tr class="odd">
-                    <td class="col-lg-1">preview</td>
-                    <td class="col-lg-2"><?php echo $oTheme->name; ?></td>
-                    <td class="col-lg-3"><?php echo $oTheme->description; ?></td>
-                    <td class="col-lg-2"><?php eT('XML themes'); ?></td>
-                    <td class="col-lg-2"><?php echo $oTheme->extends; ?></td>
-                    <td class="col-lg-1"><?php echo $oTheme->manifestbuttons; ?></td>
-                </tr>
-            <?php endforeach; ?>
-            </tbody>
-        </table>
+                    <tbody>
+                    <?php foreach ($aQuestionThemes['available_themes'] as $oTheme): ?>
+                        <tr class="odd">
+                            <td class="col-lg-1">preview</td>
+                            <td class="col-lg-2"><?php echo $oTheme->name; ?></td>
+                            <td class="col-lg-3"><?php echo $oTheme->description; ?></td>
+                            <td class="col-lg-2"><?php eT('XML themes'); ?></td>
+                            <td class="col-lg-2"><?php echo $oTheme->extends; ?></td>
+                            <td class="col-lg-1"><?php echo $oTheme->manifestbuttons; ?></td>
+                        </tr>
+                    <?php endforeach; ?>
+                    </tbody>
+                </table>
+            </div>
```

Comment:
This double div end can not move to the wrapper ?
At the same place where it start ? (in same file)

Reply:
These are clsing the ones opened in line 9
https://github.com/LimeSurvey/LimeSurvey/pull/3358/files#diff-b21a655d0b694f558c8e64c048f73126507817841d37584f482ea9a80271c38fR9

Right?

Reply:
Oh right … sorry …